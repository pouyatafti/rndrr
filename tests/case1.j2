Hello, {{ person.name }}!

How's the weather in {{ person.city }}?

I have a bag of {{ bag.of }}.  It contains:
{% for item, cnt in bag.contents|dictsort %}{{ cnt }} {{ item }}{% if cnt > 1 %}s{% endif %}{% if loop.revindex0 == 1 %}, and{% elif loop.revindex0 == 0 %}.{% else %}, {% endif %}
{% endfor %}
