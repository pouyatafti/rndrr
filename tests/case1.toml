[person]
name = "Johanna"
city = "Johannesburg"

[bag]
of = "fruit"
    [bag.contents]
    apple = 1
    orange = 3
    kiwi = 5
