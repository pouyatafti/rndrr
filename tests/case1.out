Hello, Johanna!

How's the weather in Johannesburg?

I have a bag of fruit.  It contains:
1 apple, 
5 kiwis, and
3 oranges.

